src/harness/fake_sas.py

```diff
@@ -90,6 +90,7 @@ class FakeSas(sas_interface.SasInterface):
   """
 
   def __init__(self):
+    self.maximum_batch_size = 100
```

Comment:
This is maximum number of grant requests in one DP grant request message, right ? If so, can we use a more descriptive variable name to reflect this meaning ?